base/agda/Base/Extensionality.agda

```diff
@@ -5,7 +5,7 @@ open import Relation.Binary.PropositionalEquality using (_≡_)
 
 -- Postualting extensionality is consistent with agdas underlying theory.
 postulate
-  ∀-extensionality : ∀ {ℓ ℓ′ : Level} {A : Set ℓ} {B : A → Set ℓ′} {f g : (x : A) → B x}
+  ext : ∀ {ℓ ℓ′ : Level} {A : Set ℓ} {B : A → Set ℓ′} {f g : (x : A) → B x}
```

Comment:
Why exactly did this have to be renamed?

Reply:
I did not like the old name (especially the `∀-`). I looked up some names for the postulate and `ext` seemes to be the most common one. It also fits the naming scheme of the other proof related functions, such as `refl`, `sym` and `cong`.